### PR TITLE
Fix loading of controller with nested prefix.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: src/Routing/Filter/ControllerFactoryFilter.php [hunk 1]
```diff
@@ -69,7 +69,7 @@ protected function _getController($request, $response)
             $controller = $request->params['controller'];
         }
         if (!empty($request->params['prefix'])) {
-            if (strpos('/', $request->params['prefix']) === false) {
+            if (strpos($request->params['prefix'], '/') === false) {
```
Comment (markstory) [inline src/Routing/Filter/ControllerFactoryFilter.php]: :rage4:

**Comment by markstory** [comment]
Looks good. I can give you a hand with the releases if you need them.

**Comment by ADmad** [comment]
@markstory Would be great if you could handle the releases :)

**Comment by markstory** [comment]
@ADmad Ok, I'll get them done tonight or tomorrow.

**Comment by ADmad** [comment]
Thanks!

**Pull request merged**